projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-specific-access-request/case-specific-access-request.component.html

```diff
@@ -1,16 +1,17 @@
 <exui-error-message
   *ngIf="formGroup.invalid && submitted"
   [error]="errorMessage"></exui-error-message>
-<cut-alert type="information"
-  >Authorisation is needed to access this case.<br />This could be because it's outside your jurisdiction, or you may be excluded from the case.
-    If you request access to this case, it will be logged for auditing purposes.</cut-alert>
+<cut-alert type="information">
+  {{'Authorisation is needed to access this case.' | rpxTranslate}}<br />
+  {{"This could be because it's outside your jurisdiction, or you may be excluded from the case.  If you request access to this case, it will be logged for auditing purposes." | rpxTranslate}}
```

Comment:
@DavidJayakumar - text should be in single quotes

Reply:
@Daniel-Lam the sentence already has a single quote `it's`, so we surrounded it with double quotes.

Reply:
done